code/game/objects/items/tools/weldingtool.dm

```diff
@@ -116,11 +116,12 @@
 	if(affecting && (!IS_ORGANIC_LIMB(affecting)) && user.a_intent != INTENT_HARM)
 		if(src.use_tool(H, user, 0, volume=50, amount=1))
 			if(user == H)
+				if (H in user.do_afters) return // one at a time
```

Comment:
Don't use this format of single line returns, have the return on the next line down